RiotSwiftUI/Modules/Room/PollTimeline/View/PollTimeline.swift

```diff
@@ -103,16 +60,73 @@ struct PollTimeline: View {
                 .font(theme.fonts.footnote)
                 .foregroundColor(theme.colors.tertiaryContent)
         }
-        .padding([.leading, .trailing, .top], 2.0)
+        .padding([.horizontal, .top], 2.0)
         .padding([.bottom])
     }
     
+    @ViewBuilder
+    private func answerOptionButtonFor(_ answerOption: TimelineAnswerOption) -> some View {
```

Comment:
Personally I think this would be better as a new `PollTimelineAnswerOptionButton` struct in it's own file. You could then make the `answerOptionLabelFor` function simply a `buttonLabel` property if you want to keep them separate.